modules/service/src/main/scala/lucuma/odb/graphql/predicate/GroupPredicates.scala

```diff
@@ -7,8 +7,9 @@ import grackle.Path
 import lucuma.core.model.Group
 
 class GroupPredicates(path: Path) {
-  lazy val id       = LeafPredicates[Group.Id](path / "id")
-  lazy val parentId = LeafPredicates[Group.Id](path / "parentId")
-  lazy val program  = ProgramPredicates(path / "program")
+  lazy val id        = LeafPredicates[Group.Id](path / "id")
+  lazy val parentId  = LeafPredicates[Group.Id](path / "parentId")
+  lazy val program   = ProgramPredicates(path / "program")
   lazy val existence = ExistencePredicates(path / "existence")
+  lazy val system    = ExistencePredicates(path / "system")
```

Comment:
`ExistencePredicates`?  I would have expected `LeafPredicates`, I think.  This raises the question of what we should do for these system / calibration role properties that we're adding wrt to queries.

Reply:
You're right, it has to be a `LeafPredicates` and used in the context of `IsNull`